test/e2e/proxy.go

```diff
@@ -218,15 +226,24 @@ func proxyContext(version string) {
 			defer errLock.Unlock()
 			errs = append(errs, s)
 		}
-		for i := 0; i < proxyAttempts; i++ {
-			for path, val := range expectations {
+		d := time.Since(start)
+		framework.Logf("setup took %v, starting test cases", d)
```

Comment:
Interesting. This is literally my first interactions with the ginkgo library. I'll see what it looks like when I convert text to `By()`s.